src/execution/experimental-executions.js

```diff
@@ -523,66 +679,62 @@ function getWhere(filters) {
 
 const sortToOrderBy = item => ({ column: get(item, 'element'), direction: get(item, 'sort') });
 
-const getOrderBy = (metrics, categories, type) => {
-    // For bar chart we always override sorting to sort by values (first metric)
-    if (type === 'bar' && notEmpty(metrics)) {
-        return [{
-            column: first(compact(map(metrics, 'element'))),
-            direction: 'desc'
-        }];
-    }
-
+const getOrderBy = (metrics, categories) => {
     return map(filter([...categories, ...metrics], item => item.sort), sortToOrderBy);
 };
 
-export const mdToExecutionConfiguration = (mdObj, options = {}) => {
-    const buckets = get(mdObj, 'buckets');
-    const measures = map(buckets.measures, ({ measure }) => measure);
-    const metrics = flatten(map(measures, (measure, index) => getMetricFactory(measure)(measure, buckets, index)));
-
-    let categories = getCategories(buckets);
-    let filters = getFilters(buckets);
-    if (options.removeDateItems) {
-        categories = filter(categories, ({ category }) => category.type !== 'date');
-        filters = filter(filters, item => !item.dateFilter);
+export function loadAttributesMap(projectId, categoryDisplayForms) {
```

Comment:
Please add test coverage for this method

Reply:
added basic tests